extensions/wikia/ImageReview/ImageReviewDatabaseHelper.class.php

```diff
@@ -23,24 +25,30 @@ public function selectImagesForList( $sOrder,
 		$oDB = $this->getDatawareDB( DB_SLAVE );
 
 		$oResults = $oDB->query('
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$oResults". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)